test/regression_test.sh

```diff
@@ -4,23 +4,50 @@ set -e
 
 mkdir -p results
 
-LOG_FILE=results/$TEST_NAME-$TEST_EXT.log
+LOG_FILE=$(realpath results/$TEST_NAME-$TEST_EXT.log)
```

Comment:
Not sure if you can write into the bazel-build directory, that should be read-only. You can write to `${TEST_TMPDIR}` if you want, but for the logging reasons, you can just use bazels logging it anyway does for the test.

Reply:
fixed. Backed this out and use stdout only for this logging.